### DISCUSSION TITLE
Tabs component cannot have children?

### DISCUSSION THREAD

**Discussion by shyamajp** [twilio-labs/paste#1178] [Q&A] [answered]
Hello! I am trying to see how to use Paste tabs so I took the sample code from documentation and put it in a project, but it throws an error that children does not exist inside Tabs. Is this some interface bug or am I doing something wrong?
I appreciate your help, thank you in advance!

### ERROR on <Tabs>
_Type '{ children: Element[]; baseId: string; }' is not assignable to type 'IntrinsicAttributes & TabsProps & RefAttributes<HTMLDivElement>'.
  Property 'children' does not exist on type 'IntrinsicAttributes & TabsProps & RefAttributes<HTMLDivElement>'.ts(2322)_

### SAMPLE CODE (without selectedId)
```
      <Tabs baseId='horizontal-tabs-example'>
        <TabList aria-label='My tabs'>
          <Tab>Tab 1</Tab>
          <Tab disabled>Tab 2</Tab>
          <Tab>Tab 3</Tab>
        </TabList>
        <TabPanels>
          <TabPanel>Tab 1</TabPanel>
          <TabPanel>Tab 2</TabPanel>
          <TabPanel>Tab 3</TabPanel>
        </TabPanels>
      </Tabs>
```

**Comment by SiTaggart** [comment, marked as answer]
Hi @shyamajp, What version of Paste are you currently using?

A bug in the Interface was fixed and released in `@twilio-paste/core@4.3.1`

**Reply by shyamajp** [reply]
Thank you!
